next.config.js

```diff
@@ -1,11 +1,8 @@
 /** @type {import('next').NextConfig} */
 
 const path = require("path");
-module.exports = {
-  sassOptions: {
-    includePaths: [path.join(__dirname, "styles")],
-  },
-};
```

Comment:
is this going to break our ability to use styles file? I know we are all in on tailwind, but we still have a bit in styles, like custom fonts and some things that are challenging to style via tailwind